cdhweb/people/urls.py

```diff
@@ -9,9 +10,10 @@
     url(r'^staff/$', views.StaffListView.as_view(), name='staff'),
     url(r'^postdocs/$', views.PostdocListView.as_view(), name='postdocs'),
     url(r'^students/$', views.StudentListView.as_view(), name='students'),
-    url(r'^faculty/$', views.FacultyListView.as_view(), name='faculty'),
+    url(r'^affiliates/$', views.AffiliateListView.as_view(), name='affiliates'),
     url(r'^speakers/$', views.SpeakerListView.as_view(), name='speakers'),
     url(r'^executive-committee/$', views.ExecListView.as_view(), name='exec-committee'),
-
+    # redirect from /people/faculty -> /people/affiliates
+    url(r'^faculty/$', RedirectView.as_view(url='/people/affiliates', permanent=True)),
```

Comment:
Nice. Should include the trailing slash to avoid redirecting twice.

Reply:
good catch, thx